routers/web/repo/view.go

```diff
@@ -977,6 +977,18 @@ func renderCode(ctx *context.Context) {
 		return
 	}
 
+	if ctx.Doer != nil {
+		if err := ctx.Repo.Repository.GetBaseRepo(ctx); err != nil {
```

Comment:
That's not a `LoadBaseRepo` yet?

Reply:
> 

It's a name inconsistent method, but not related with this PR.